bundles/org.openhab.binding.snmp/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -106,6 +102,7 @@
 					<option value="AES192">AES192</option>
 					<option value="AES256">AES256</option>
 					<option value="DES">DES</option>
+					<option value="DES3">3DES</option>
```

Comment:
Changes to this file would als require changes to the snmp.properties file

Reply:
Similar to https://github.com/openhab/openhab-addons/pull/16839#discussion_r1623569622 it would be better to not include them at all. But I agree that it's inconsistent to only have four of them, and since they will reappear when running `mvn i18n:generate-default-translations` I guess the best short-term solution is to add it/regenerate.